src/components/sdk/MeetingControls/AudioInputVFControl.tsx

```diff
@@ -128,47 +115,55 @@ const AudioInputVFControl: React.FC<Props> = ({
         console.error(
           'AudioInputVFControl failed to select audio input device'
         );
+      } finally {
+        setIsLoading(false);
```

Comment:
I think we dont want any `isLoading` state update changes in the `else` block under try. May be just keep it as is, not sure yet if that impacts anything.

Reply:
Nope. if `await selectAudioInput(currentDevice);` throws the error `setIsLoading(false);` can not be reached and thus add it to `finally`.

Maybe I can remove line 110 to avoid duplicate, how do you think?